exporter/instanaexporter/exporter.go

```diff
@@ -112,6 +115,32 @@ func (e *instanaExporter) export(ctx context.Context, url string, header map[str
 		return consumererror.NewPermanent(err)
 	}
 
+	if e.config.CAFile != "" {
+		caCertPool := x509.NewCertPool()
```

Comment:
Would it be better to assign this after the file is read successfully? Otherwise it is not required.